eng/AfterSolutionBuild.targets

```diff
@@ -0,0 +1,3 @@
+<Project>
+    <Import Project="$(MSBuildThisFileDirectory)WpfArcadeSdk\tools\RunDrtsLocal.targets" Condition="'$(Test)'=='True'" />
```

Comment:
The WPF Arcade SDK defines path vars that can be used to plumb down to the target you want.  It seems to me that if you used `WpfArcadeSdkToolsDir` you could move this targets file down into the SDK itself without an issue.  Then, if you do the props stuff I was talking about elsewhere, only the props file requires syncing across repos.  Well, apart from DARC deps.

Reply:
Bah!  Misunderstood this entirely!  Nevermind about moving things down.